src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,75 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinTable(name = "sessions_movies",
+            joinColumns = @JoinColumn(name = "movie_session_id"),
+            inverseJoinColumns = @JoinColumn(name = "movie_id"))
+    private Movie movie;
+    @ManyToOne
+    @JoinTable(name = "sessions_halls",

```

Comment:
For many-to-one relation it's surely possible to use jointable but having just column with foreign key is more simple solution and fully suitable here